lib/ansible/modules/windows/win_auto_logon.ps1

```diff
@@ -9,75 +9,75 @@
 
 Set-StrictMode -Version 2.0
 
-$ErrorActionPreferrence = 'Stop'
-
-# argument spec hastable which has the input parameters and capabilities for this module
 $spec = @{
     options = @{
-        domain = @{type = "str"; default = $env:userdomain}
-        user = @{type = "str"; required = $true}
-        password = @{type = "str"; required = $true}
+        password = @{type = "str"; no_log = $true}
```

Comment:
Sensitive fields should have `no_log = $true` set to avoid it being accidentally displayed on the module invocation return value.